### ISSUE TITLE
Dracula theme for tmux

### ISSUE THREAD

**Issue by sei40kr** [dracula/dracula-theme#240]
# Dracula theme for tmux

##  tmux-airline-dracula

https://github.com/sei40kr/tmux-airline-dracula

![screenshot](https://user-images.githubusercontent.com/11665236/29017752-2bff3e58-7b93-11e7-97c7-4ab23092f904.png)

**Comment by pichardoJ** [comment]
That's awesome dude, here is mine, hope you like it
![2017-10-27-194728_935x985_scrot](https://user-images.githubusercontent.com/12650088/32130586-d135ef38-bb4f-11e7-87a5-c0e5fb7d56a4.png)

**Comment by mamyn0va** [comment]
![](https://thepracticaldev.s3.amazonaws.com/i/cn3qgitf5xx7kr223adt.png)

[](https://mamyn0va.github.io/2019/08/14/consistent-developer-journey-with-dracula)

**Comment by hacknug** [comment]
Any of you would like to thansfer your theme to the organization so its easier for interested users to find it?

**Comment by mamyn0va** [comment]
Why not, but why do you mean by "transfer your theme to the organization" ?

**Comment by hacknug** [comment]
@pichardoJ since OP's repo has been archived, do you have a repo with your version of the theme? If not would you mind creating one using the template if @mamyn0va doesn't have one either? You could also use @sei40kr's if the instructions are still correct as long as we update the screenshot to show a little more.

**Comment by mamyn0va** [comment]
My tmux dracula theme is forked from another tmux conf (not dracula), so I guess I need to ask the author if I can transfer my theme to your organization, under MIT LICENSE.
@gpakosz What do you think?

**Comment by dagadbm** [comment]
any news on this?

**Comment by mamyn0va** [comment]
I would be happy to provide my theme but I still need the approval from @gpakosz.
In the meanwhile, you can find it here: https://github.com/mamyn0va/tmux

**Comment by danerwilliams** [comment]
https://github.com/danerwilliams/tmux-dracula

Only tested my theme on mac os so far. I need to look in to how my unix pipelines for getting battery information and network SSID might change from mac to linux.

![image](https://user-images.githubusercontent.com/22798229/78613343-74145e80-7820-11ea-9baf-33121c739cae.png)

Install with tpm by adding `set -g @plugin 'danerwilliams/tmux-dracula'` to .tmux.conf if anyone wants to try it out :)

Feedback is welcome!

**Comment by zenorocha** [comment]
It took a while, but now it's official:
* https://draculatheme.com/tmux
* https://github.com/dracula/tmux

I would love to hear your thoughts on this @mamyn0va @pichardoJ @sei40kr

**Comment by dagadbm** [comment]
nice!

i just wish there was an easier way to have tmux themes with customization
on the  bottom bar. For example the one dark tmux theme has those powerline
symbols and so on and I always wanted to know how to have that more custom
but with the color scheme maintaining and it seems the two are very tightly
coupled right?

**Comment by danerwilliams** [comment]
@dagadbm I think that it's possible to set it up so that powerline symbols would be an optional setting. It looks like  [nord theme tmux](https://github.com/arcticicestudio/nord-tmux/blob/develop/nord.tmux) does this. 

if [ "$status_content" != "0" ]; then
    if [ "$no_patched_font" != "1" ]; then
      tmux source-file "$_current_dir/$NORD_TMUX_STATUS_CONTENT_FILE"
    else
      tmux source-file "$_current_dir/$NORD_TMUX_STATUS_CONTENT_NO_PATCHED_FONT_FILE"
    fi
 fi

Maybe this would be a good issue / feature request to add?

**Comment by atgmello** [comment]
Not sure if this is the appropriate place, but it's about Dracula and tmux, so I guess it fits.

After checking this recent release I realized that my tmux needed a revamp to match my whole Dracula setup. However, I'm already using the great [Oh My Tmux](https://github.com/gpakosz/.tmux) configuration file. Inspired by this release and Dracula's Vim Airline theme, I decided setting up a minimal "theme" for Oh My Tmux.

So that's what I did!: 
https://github.com/atgmello/dracula.omt

![image](https://user-images.githubusercontent.com/4420461/79346909-4fe40c00-7f09-11ea-8d86-2cf8e1d845d7.png)

I'd note that @mamyn0va 's effort is appreciated. However I'd like to keep my version up to date with @gpakosz 's repo and also keep my other configuration intact. I guess that what I came up with solves both issues.

I would also like to take this opportunity to thank the whole Dracula Team! You're doing an amazing job porting such an awesome theme for so many apps! :)
